modules/ROOT/pages/_partials/mule-runtime-metrics.adoc

```diff
@@ -46,41 +46,62 @@ The maximum number of concurrent flows across all Mule apps and business groups
 
-Deployment:: 
-The runtime plane to which the Mule app is deployed. CloudHub (abbreviated as CH), CloudHub 2.0 (abbreviated as CH2), and Runtime Fabric (abbreviated as RTF) are supported. 
+| Business Group | The business group a Mule app is deployed to.
 
-Flows:: 
-The sum of flows in the Mule app. This number is calculated by multiplying flows by the number of workers (CloudHub) or replicas (CloudHub 2.0). For more information about how this metric is billed, see xref:usage-metrics.adoc#mule-flows[Mule flows].
+| Deployment Type | The runtime plane to which the Mule app is deployed. CloudHub (abbreviated as CH), CloudHub 2.0 (abbreviated as CH2), and Runtime Fabric (abbreviated as RTF) are supported. 
 
-Messages::
-The sum of inbound and outbound Mule messages in the Mule app. For more information about how this metric is billed, see xref:usage-metrics.adoc#mule-messages[Mule messages].  
+|Environment Type |The environment in which a Mule app is deployed.
 
-Data Throughput::
-The sum of inbound and outbound data in gigabytes (GB) transmitted by the Mule app. For more information about how this metric is billed, see xref:usage-metrics.adoc#data-throughput[data throughput].  
+| Cluster | The name of the cluster containing the replicas and worker nodes.
+
+| CPU Limit | The maximum amount of vCPU the application can use. This is shared CPU on the worker node.
+
+| CPU Reserve (Millicores) | The amount of vCPU guaranteed to the application and reserved for its use.
+
+| Mule Flows | The sum of flows in the Mule app. This number is calculated by multiplying flows by the number of workers (CloudHub) or replicas (CloudHub 2.0). For more information about how this metric is billed, see xref:usage-metrics.adoc#mule-flows[Mule flows].
+
+| Mule Messages | The sum of inbound and outbound Mule messages in the Mule app. For more information about how this metric is billed, see xref:usage-metrics.adoc#mule-messages[Mule messages].  
+
+| Data Throughput (GB) | The sum of inbound and outbound data in gigabytes (GB) transmitted by the Mule app. For more information about how this metric is billed, see xref:usage-metrics.adoc#data-throughput[data throughput].  
+|===
 
-Number of Workers::
-The sum of all workers (CloudHub) or replicas (CloudHub 2.0) of a Mule app. Apps that don’t have workers or replicas show a value of `N/A` in this column.
 // end::table[]
 
 // tag::flows[]
-== Mule Flows
+=== Mule Flows
+
+A Mule flow is a sequence of logical operations configured within the XML `<flow/>` element of a Mule application. The runtime report tracks the Mule flows within a deployed and running Mule application that contains a Mule event source or APIkit route requests. Mule apps in production environments typically use multiple Mule flows and subflows to divide the app into functional modules or for error-handling purposes. For example, one Mule flow might receive a record and transform data into a given format that another flow processes.  
```

Comment:
Is it possible for apps to be deployed and not running (I forget)?

Can you split this para at Mule apps?

Reply:
Yes, apps can be deployed and not running.

Reply:
This partial isn't being used, so I'm fixing in the usage-metrics file